docs/man/borg.1

```diff
@@ -358,6 +358,9 @@ passphrase should be initially set when initializing an encrypted repo.
 If BORG_PASSPHRASE is also set, it takes precedence.
 See also BORG_NEW_PASSPHRASE.
 .TP
+.B BORG_PASSPHRASE_FD
+When set, specifies a file descriptor to read a passphrase from. Programs starting borg may choose to open an anonymous pipe and use it to pass a passphrase. This is safer than passing via BORG_PASSPHRASE, because on some systems (e.g. Linux) environment can be examined by other processes.
+.TP
```

Comment:
if this was automatically generated, you do not need to include it in the PR.

but you need to include the change you (hopefully) did to the `.rst` or `.py` file which is the stuff one needs to edit.

all docs / man pages / etc. will be autogenerated as part of the release process.

Reply:
Moved to docs/usage_general.rst.inc